docs/how-to-guides/networkx.md

```diff
@@ -5,10 +5,134 @@ sidebar_label: Utilize the NetworkX library
+
+This how-to guide will show one simple example of calling a NetworkX procedure. 
+
+### 1. Connect to Memgraph
+
+To be able to try out this example on your own, you need a running Memgraph instance. Run Memgraph using the Memgraph Platform Docker image, which includes both the MAGE library and Memgraph Lab, a visual interface. 
+To run the image, open a command-line interpreter and run the following Docker command:
+
+```
+docker run -it -p 7687:7687 -p 7444:7444 -p 3000:3000 memgraph/memgraph-platform:latest
+```
+
+[Connect to Memgraph](/docs/memgraph-lab/connect-to-memgraph#connecting-to-memgraph) via Memgraph Lab which is running at `localhost:3000`. 
+
+Check out the [installation guide](/docs/memgraph/installation) for other installation options. If you wish to avoid the installation, you can also use [Memgraph Cloud](/docs/memgraph-cloud/).
+
+### 2. Load the dataset
+
+First, head over to the **Datasets** section to load a dataset. Then, choose the **Europe backpacking dataset**. 
+
+<img src={require('../data/how-to-guides/nxalg-how-to-1.png').default} className={"imgBorder"}/>
+
+### 3. Run NetworkX algorithm
+
+Once the dataset is loaded, go to the **Query Modules** section and search for `nxalg` module. Click on the arrow next to the module name to **view module details**. 
+
+<img src={require('../data/how-to-guides/nxalg-how-to-2.png').default} className={"imgBorder"}/>
+
+The goal is to run the [`is_bipartite()`](/docs/mage/query-modules/python/nxalg#is_bipartite) procedure to check whether the graph is bipartite. 
+
+<img src={require('../data/how-to-guides/nxalg-how-to-3.png').default} className={"imgBorder"}/>
+
+Copy the query, go to the **Query Execution** tab and paste the query into the **Cypher Editor**. If you need the query, here it is:
+
+```cypher 
+CALL nxalg.is_bipartite() YIELD is_bipartite;
+```
+
+By clicking on the **Run Query** button, you can see that the Europe backpacking graph is not bipartite.
+
+<img src={require('../data/how-to-guides/nxalg-how-to-4.png').default} className={"imgBorder"}/>
+
+In the same way, you can run other procedures from the `nxalg` module and the procedures from the `graph_analyzer` module, which can be found in the **Query Modules** section.
+
+## How to implement custom NetworkX module
+
+Besides using already implemented modules, you can create your own module which utilizes the NetworkX library.
+To learn how to implement a custom query module, head over to the [example of query module in Python](/docs/memgraph/reference-guide/query-modules/implement-custom-query-modules/custom-query-module-example#python-api).
+
+Since Memgraph is integrated with NetworkX, you can import NetworkX library inside your Python code and start coding. In this guide, we will create a new query module which utilizes the NetworkX library with the help of Memgraph Lab.
```

Comment:
```suggestion
Since Memgraph is integrated with NetworkX, you can import NetworkX library inside Python code. This guide will show you how to create a new query module that utilizes the NetworkX library within Memgraph Lab.
```